clients/fossid-webapp/src/main/kotlin/FossIdRestService.kt

```diff
@@ -58,8 +60,13 @@ import retrofit2.Retrofit
 import retrofit2.converter.jackson.JacksonConverterFactory
 import retrofit2.http.Body
 import retrofit2.http.GET
+import retrofit2.http.Headers
 import retrofit2.http.POST
 
+private const val READ_TIMEOUT_HEADER = "READ_TIMEOUT"
+private const val CONNECT_TIMEOUT_HEADER = "CONNECT_TIMEOUT"
+private const val WRITE_TIMEOUT_HEADER = "WRITE_TIMEOUT"
+
```

Comment:
IIUC you have the intention to only change the READ_TIMEOUT with this PR.
Put the implementation has logic also for changing CONNECT_TIMEOUT, which seems unused even after the following commit. Would it make sense to remove that?

Reply:
Yes currently I only use the `READ_TIMEOUT `. But since one can configure all the three with the header, I thought it would make sense to have them all configurable.

Reply:
> But since one can configure all the three with the header, I thought it would make sense to have them all configurable.

I'd prefer to keep things simple here and not make these configurable unless needed, esp. as the code otherwise contains some not-so-nice duplications.